spec/xrootd-cmstfc.spec

```diff
@@ -11,9 +10,28 @@ URL: https://github.com/bbockelm/xrootd-cmstfc
 # git-archive master | gzip -7 > ~/rpmbuild/SOURCES/xrootd-lcmaps.tar.gz
 Source0: %{name}.tar.gz
 BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-BuildRequires: xrootd-libs-devel xerces-c-devel pcre-devel
+
+%define xrootd_current_major 4
+%define xrootd_next_major 5
+
+BuildRequires: xrootd-devel >= 1:%{xrootd_current_major}.0.0-0
+BuildRequires: xrootd-devel <  1:%{xrootd_next_major}.0.0-0
+BuildRequires: pcre-devel
+
+BuildRequires: xerces-c-devel
+
 BuildRequires: cmake
+#BuildRequires: xrootd-compat-libs
+
 Requires: /usr/bin/xrootd pcre xerces-c
+#Requires: xrootd-compat-libs
+
+#%if 0%{?rhel} < 7
+#Requires: xrootd4 >= 1:4.1.0
+#%else
+Requires: xrootd >= 1:%{xrootd_current_major}.0.0-0
```

Comment:
This isn't correct.  Xrootd will fail to load if its runtime major.minor is older than the _build-time_ major.minor.

- *OK*: building with 4.0 and running with 4.10.
- *NOT OK*: building with 4.12 and running with 4.10.

Reply:
For what we discuss lately with Andy, the ABI compatibility shouldn't be broken among the major version of xrootd, i.e. a plugin build against 4 should work with any version of 4 but not with 5. This is contrary to what we use to believe in OSG

Reply:
Not quite - see this code:

https://github.com/xrootd/xrootd/blob/master/src/XrdSys/XrdSysPlugin.cc#L204-L208

Reply:
changes applied